### PR TITLE
Character limit per line

### PR DESCRIPTION
**Pull request by siuwuncheung**
This PR simply adds a 80-character limit to the scripts throughout libROM using astyle, similar to MFEM. Short lines are more easily read and reviewed on GitHub.

Long lines can be truncated by running `./scripts/stylize.sh $PATH_TO_ASTYLE`.

### CODE REVIEW

**Review by kevinhkhuynh — APPROVED** [review]

**Comment by kevinhkhuynh** [comment]
Can we wait until S-OPT is merged before merging this? I feel like this change will cause a bunch of merge conflicts.

**Comment by dylan-copeland** [comment]
Since we are already changing a lot of files, some other simple changes could be made in this PR:
1. Update copyright year from 2021 to 2022 (multiple files).
2. Change .C to .cpp.

**Review by chldkdtn — APPROVED** [review]

**Comment by kevinhkhuynh** [comment]
@dylan-copeland Requested updates have been made.

**Pull request merged**
